_posts/2018-03-29-Rust-1.25.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: post
+title: "Announcing Rust 1.25"
+author: The Rust Core Team
+---
+
+The Rust team is happy to announce a new version of Rust, 1.25.0. Rust is a
+systems programming language focused on safety, speed, and concurrency.
+
+If you have a previous version of Rust installed via rustup, getting Rust
+1.25.0 is as easy as:
+
+```bash
+$ rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`][install] from the
+appropriate page on our website, and check out the [detailed release notes for
+1.25.0][notes] on GitHub.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1250-2018-03-29
+
+## What's in 1.25.0 stable
+
+Rust 1.25 contains a bunch of stuff! The first one is straightforward: we've
+[upgraded to LLVM 6] from LLVM 4. This has a number of effects, a major one
+being a step closer to AVR support.
+
+A new way to write `use` statements has landed: [nested import groups]. If you've
+ever written a set of imports like this:
+
+```rust
+use std::fs::File;
+use std::io::Read;
+use std::path::{Path, PathBuf};
+```
+
+You can now write this:
+
+```rust
+// on one line
+use std::{fs::File, io::Read, path::{Path, PathBuf}};
+
+// with some more breathing room
+use std::{
+    fs::File,
+    io::Read,
+    path::{
+        Path,
+        PathBuf
+    }
+};
+```
+
+This can reduce some repetition, and make things a bit more clear.
+
+There are two big documentation changes in this release: first, [Rust By
+Example is now included on doc.rust-lang.org]! We'll be redirecing the old
+domain there shortly. We hope this will bring more attention to a great
+resource, and you'll get a local copy with your local documentation.
+
+Second, back in Rust 1.23, we talked about the change from Hoedown to
+pulldown-cmark. In Rust 1.25, pulldown-cmark is now the default rendering.
+After years, we have finally removed the last bit of C from rustdoc, and
```

Comment:
This might be correct, but "After years" sounds a little odd -- maybe remove entirely? I can't think of what would go well there...

Reply:
sure, if you think it is. idk.